buildpacks/builder.go

```diff
@@ -100,3 +124,466 @@ func (builder *Builder) Build(ctx context.Context, f bosonFunc.Function) (err er
 
 	return
 }
+
+type clientWrapper struct {
+	impl dockerClient.CommonAPIClient
+}
+
+// override injecting the security opt
+
+func (c clientWrapper) ContainerCreate(ctx context.Context, config *container.Config, hostConfig *container.HostConfig, networkingConfig *network.NetworkingConfig, platform *v1.Platform, containerName string) (container.ContainerCreateCreatedBody, error) {
+	hostConfig.SecurityOpt = []string{"label=disable"}
+	return c.impl.ContainerCreate(ctx, config, hostConfig, networkingConfig, platform, containerName)
+}
+
+// just plain forwards
+
+func (c clientWrapper) ConfigList(ctx context.Context, options types.ConfigListOptions) ([]swarm.Config, error) {
```

Comment:
Is there any sane way as to do this? Why can't I just override the function I want to. :cry: